### PR TITLE
Translate App Strings to Japanese

### LINKED ISSUE #2352: Translate App Strings to Japanese
### User Story

```Please note: we have no speakers of this language on the team, so it's difficult to check the validity of translations. Therefore, on issues like this, we will wait for at least 2 PRs before merging anything (within a sane timeframe). If you submit a PR, please also justify why your translation is better than any that have previously been submitted, so we can make an informed choice about which to merge. In the case that we need to use elements of both, we will most likely reward bounties to both contributors, for making and verifying the translations.```

As a Japanese language speaker, I want to be able to interact with Status in my native language.

### Description

*Type*: Feature

*Summary*: We need someone fluent (and eloquent) in Japanese to translate all the latest strings in Status from English. We would like to invite our community members to take on these tasks in order
a) To become a bit more acquainted with the Status repos and perhaps wander further into the codebase and set of issues we need help with, earning yourself Open Bounties all the way down this rabbit hole, and
b) To make sure that the product we deliver has been built by the same community that uses it, in a virtuous feedback loop of increasing understanding and value-generation.

#### Expected behavior
The Japanese translation ought to be as up-to-date as the English one.

#### Actual behavior
We have not yet updated it, preferring to leave this open as an opportunity for our community to get involved again.

### Solution

*Summary*: 

- The file [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/ja.cljs) needs to be updated to match the new English one [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/en.cljs).
- How you translate is a matter of your preference. There were significant updates to the English file for the 0.9.12 release, with lots of strings both add and REMOVED. We need you to make sure that the files are exact mirrors of each other, except in different languages.
- It might be easier to simply take the new English file and translate from scratch rather than going through the current translated files line by line (though we leave this up to you).

### Additional Information

* Status version: 0.9.12+

There are bounties available for these translations. Please see the QR codes that will be added below after the DevCon 3 announcement.

### PR DESCRIPTION
**Pull request by dumiz**
fixes #2352

### Summary:
Translate app strings to Japanese.
Some mistakes in old Japanese translation, fixed them as well.

status: ready

### CODE REVIEW

File: src/status_im/translations/ja.cljs [hunk 1]
```diff
@@ -11,12 +11,19 @@
...
    :type-a-command                        "コマンドを入力する..."
    :error                                 "エラー"
+   :unknown-status-go-error               "不明なstatus-goがエラー"
+   :node-unavailable                      "実行中のエーテル　ノードがありません"
+   :yes                                   "はい"
+   :no                                    "いいえ"
 
-   :camera-access-error                   "カメラを利用できません。設定よりカメラの使用を許可してください。"
-   :photos-access-error                   "写真を取得できません。設定より写真へのアクセスを許可してください。"
+   :camera-access-error                   "必要なカメラの許可を与えるには、システム設定に行き、ステータス>カメラが選択されていることを確認してください。""
```
Comment (andytudhope) [inline src/status_im/translations/ja.cljs]: Remove extra `"` - it breaks everything

File: src/status_im/translations/ja.cljs [hunk 2]
```diff
@@ -158,83 +177,204 @@
...
+   :transactions-history-empty            "あなたの履歴にまだ取引はありません"
+   :transactions-unsigned-empty           "署名のない取引はありません"
+   :transactions-filter-title             "フィルタ履歴"
+   :transactions-filter-tokens            "トークン"
+   :transactions-filter-type              "タイプ"
+   :transactions-filter-select-all        "全て選択"
+   :view-transaction-details              "取引の詳細を表示する"
+   :transaction-description               "あなたの取引が安全に処理されることを確認するために少なくとも12回の確認を待ってください。"
+   :transaction-sent                      "送信された取引"
+   :transaction-moved-text                "取引は "未署名"リストに５分間を残ります"
```
Comment (andytudhope) [inline src/status_im/translations/ja.cljs]: Change the quoted text to use `'` single quotes, otherwise it breaks everything.

**Review by andytudhope — CHANGES_REQUESTED** [review]
In addition to the two changes below we will need 2 other things for this PR to be accepted.

1) Please edit the file `i18n.js` and add the key `:ja` into this function: https://github.com/status-im/status-react/blob/develop/src/status_im/i18n.cljs#L270
Then, you can run `lein test-cljs` and see for yourself that you have indeed translated all the strings we require (working off the English file). Nice work.

2) We need to find another Japanese reviewer. Do you have a friend with a decent GitHub profile that could review your translations? We will also tweet it out from our official account to see if we can find anyone who can do it.

File: src/status_im/translations/ja.cljs [hunk 3]
```diff
@@ -158,83 +177,204 @@
...
+   :request-transaction                   "取引をリクエストする"
+   :send-request                          "リクエストを送信する"
+   :share                                 "シェア"
+   :eth                                   "ETH"
+   :currency                              "通貨"
+   :usd-currency                          "USD"
+   :transactions                          "取引"
+   :transaction-details                   "取引の詳細"
+   :transaction-failed                    "取引が失敗しました"
+   :transactions-sign                     "サイン"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "サイン" → "署名"

File: src/status_im/translations/ja.cljs [hunk 4]
```diff
@@ -158,83 +177,204 @@
...
+   :transactions-sign-all                 "すべてにサインする"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "すべてにサインする" → "全てに署名する"

File: src/status_im/translations/ja.cljs [hunk 5]
```diff
@@ -158,83 +177,204 @@
...
+   :transactions-sign-transaction         "取引に署名する"
+   :transactions-sign-later               "後でサインする"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "後でサインする" → "後で署名する"

File: src/status_im/translations/ja.cljs [hunk 6]
```diff
@@ -158,83 +177,204 @@
...
+   :transactions-delete                   "取引を削除する"
+   :transactions-delete-content           "取引は「未署名」リストから削除されます。"
+   :transactions-history                  "歴史"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "歴史" → "取引履歴"

File: src/status_im/translations/ja.cljs [hunk 7]
```diff
@@ -158,83 +177,204 @@
...
    :removed-from-chat                     "グループチャットから削除されました"
    :left                                  "残り"
    :invited                               "招待済み"
    :removed                               "削除済み"
    :You                                   "あなた"
 
-   ;new-contact
+   ;;new-contact
    :add-new-contact                       "新規連絡先を追加"
    :scan-qr                               "QRをスキャン"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "QRをスキャン" → "ORコードをスキャン"

File: src/status_im/translations/ja.cljs [hunk 8]
```diff
@@ -158,83 +177,204 @@
...
    :recipient                             "受信者"
    :to                                    "宛先"
    :from                                  "送信元"
    :data                                  "データ"
    :got-it                                "理解しました"
+   :block                                 "ブロック"
+   :hash                                  "ハッシュ"
+   :gas-limit                             "gas制限"
+   :gas-price                             "gas価格"
+   :gas-used                              "Gas used"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "Gas used" → "使用gas"

File: src/status_im/translations/ja.cljs [hunk 9]
```diff
@@ -158,83 +177,204 @@
...
+   :confirmations-helper-text             "あなたの取引が安全に処理されることを確認するために少なくとも12回の確認を待ってください。"
+   :copy-transaction-hash                 "トランザクションハッシュをコピーする"
+   :open-on-etherscan                     "Etherscan.ioを開く"
+   :incoming                              "着信"
+   :outgoing                              "送信"
+   :pending                               "未定"
+   :postponed                             "延期された"
+
+   ;;webview
+   :web-view-error                         "おっと、エラーが発生しました。"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "おっと、エラーが発生しました。" → "エラーが発生しました"

File: src/status_im/translations/ja.cljs [hunk 10]
```diff
@@ -158,83 +177,204 @@
...
+
+   ;;testfairy warning
+   :testfairy-title                       "警告!"
+   :testfairy-message                     "あなたはインストールされたアプリは夜間のバージョンです。 テスト目的のために、このバージョンにWi-Fi接続が使用されている場合は会話記録が含まれています。このアプリとのすべてのやり取りは（動画やログとして）保存され、開発チームが問題の調査に使用する可能性があります。 保存されたビデオ/ログにはパスワードは含まれません。 会話記録は、アプリが夜間のバージョンからインストールされている場合にのみ実行されます。 アプリがPlayStoreまたはTestFlightからインストールされている場合は記録されません。"
+
+   ;; wallet
+   :wallet                                "財布"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "財布" → "ウォレット"

File: src/status_im/translations/ja.cljs [hunk 11]
```diff
@@ -158,83 +177,204 @@
...
+   :wallets                               "全ての財布"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "全ての財布" → "全てのウォレット"

File: src/status_im/translations/ja.cljs [hunk 12]
```diff
@@ -158,83 +177,204 @@
...
+   :your-wallets                          "あなたの全ての財布"
```
Comment (fox-daiki-yamamoto) [inline src/status_im/translations/ja.cljs]: "あなたの全ての財布" → "あなたの全てのウォレット"

File: src/status_im/i18n.cljs [hunk 13]
```diff
@@ -284,7 +284,8 @@
                                                 :lt
                                                 :sr-RS_#Latn
                                                 :sr
-                                                :sv}))
+                                                :sv
+												:ja}))
```
Comment (andytudhope) [inline src/status_im/i18n.cljs]: please align correctly here

**Comment by andytudhope** [comment]
@fox-daiki-yamamoto are you in our https://chat.status.im ? Ping me @cryptowanderer in there if you are?

**Comment by dumiz** [comment]
@andytudhope Thanks!
Sorry, I am not really a coder, I don't have friends using GitHub. So I guess you would probably need to find someone as a reviewer.
Is @fox-daiki-yamamoto the reviewer you find?

Also, when I was trying to run `lein test-cljs` I got the following errors:
```
clojure.lang.ExceptionInfo: failed compiling file:src\status_im\utils\js_resources.cljs {:file
#object[java.io.File 0x76a41af0 "src\\status_im\\utils\\js_resources.cljs"]}
        at clojure.core$ex_info.invokeStatic(core.clj:4739)
        at clojure.core$ex_info.invoke(core.clj:4739)
        at cljs.compiler$compile_file$fn__4585.invoke(compiler.cljc:1552)
        at cljs.compiler$compile_file.invokeStatic(compiler.cljc:1513)
        at cljs.compiler$compile_file.invoke(compiler.cljc:1489)
        at cljs.closure$compile_file.invokeStatic(closure.clj:540)
        at cljs.closure$compile_file.invoke(closure.clj:531)
        at cljs.closure$eval6840$fn__6841.invoke(closure.clj:609)
        at cljs.closure$eval6776$fn__6777$G__6765__6784.invoke(closure.clj:493)
        at cljs.closure$compile_sources$iter__6964__6968$fn__6969.invoke(closure.clj:954)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.Cons.next(Cons.java:39)
        at clojure.lang.RT.next(RT.java:706)
        at clojure.core$next__5108.invokeStatic(core.clj:64)
        at clojure.core$dorun.invokeStatic(core.clj:3134)
        at clojure.core$doall.invokeStatic(core.clj:3140)
        at clojure.core$doall.invoke(core.clj:3140)
        at cljs.closure$compile_sources.invokeStatic(closure.clj:950)
        at cljs.closure$compile_sources.invoke(closure.clj:939)
        at cljs.closure$build.invokeStatic(closure.clj:2596)
        at cljs.closure$build.invoke(closure.clj:2507)
        at cljs.build.api$build.invokeStatic(api.clj:205)
        at cljs.build.api$build.invoke(api.clj:189)
        at cljs.build.api$build.invokeStatic(api.clj:192)
        at cljs.build.api$build.invoke(api.clj:189)
        at user$eval7910.invokeStatic(form-init3314955066129883096.clj:1)
        at user$eval7910.invoke(form-init3314955066129883096.clj:1)
        at clojure.lang.Compiler.eval(Compiler.java:7062)
        at clojure.lang.Compiler.eval(Compiler.java:7052)
        at clojure.lang.Compiler.load(Compiler.java:7514)
        at clojure.lang.Compiler.loadFile(Compiler.java:7452)
        at clojure.main$load_script.invokeStatic(main.clj:278)
        at clojure.main$init_opt.invokeStatic(main.clj:280)
        at clojure.main$init_opt.invoke(main.clj:280)
        at clojure.main$initialize.invokeStatic(main.clj:311)
        at clojure.main$null_opt.invokeStatic(main.clj:345)
        at clojure.main$null_opt.invoke(main.clj:342)
        at clojure.main$main.invokeStatic(main.clj:424)
        at clojure.main$main.doInvoke(main.clj:387)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.lang.Var.applyTo(Var.java:702)
        at clojure.main.main(main.java:37)
Caused by: clojure.lang.ExceptionInfo: node_modules\web3\dist\web3.min.js (The system cannot find the path specified) at line 44 src\status_im\utils\js_resources.cljs {:file "src\\status_im\\utils\\js_resources.cljs", :line 44, :column 16, :tag :cljs/analysis-error}
        at clojure.core$ex_info.invokeStatic(core.clj:4739)
        at clojure.core$ex_info.invoke(core.clj:4739)
        at cljs.analyzer$error.invokeStatic(analyzer.cljc:697)
        at cljs.analyzer$error.invoke(analyzer.cljc:693)
        at cljs.analyzer$macroexpand_1.invokeStatic(analyzer.cljc:3333)
        at cljs.analyzer$macroexpand_1.invoke(analyzer.cljc:3329)
        at cljs.analyzer$analyze_seq.invokeStatic(analyzer.cljc:3366)
        at cljs.analyzer$analyze_seq.invoke(analyzer.cljc:3346)
        at cljs.analyzer$analyze_form.invokeStatic(analyzer.cljc:3536)
        at cljs.analyzer$analyze_form.invoke(analyzer.cljc:3532)
        at cljs.analyzer$analyze_STAR_.invokeStatic(analyzer.cljc:3585)
        at cljs.analyzer$analyze_STAR_.invoke(analyzer.cljc:3576)
        at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:3610)
        at cljs.analyzer$analyze.invoke(analyzer.cljc:3593)
        at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:3602)
        at cljs.analyzer$analyze.invoke(analyzer.cljc:3593)
        at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:3600)
        at cljs.analyzer$analyze.invoke(analyzer.cljc:3593)
        at cljs.analyzer$analyze_js_star_STAR_$fn__3075.invoke(analyzer.cljc:3028)
        at clojure.core$map$fn__5587.invoke(core.clj:2747)
        at clojure.lang.LazySeq.sval(LazySeq.java:40)
        at clojure.lang.LazySeq.seq(LazySeq.java:49)
        at clojure.lang.Cons.next(Cons.java:39)
        at clojure.lang.PersistentVector.create(PersistentVector.java:73)
        at clojure.lang.LazilyPersistentVector.create(LazilyPersistentVector.java:44)
        at clojure.core$vec.invokeStatic(core.clj:377)
        at clojure.core$vec.invoke(core.clj:367)
        at cljs.analyzer$analyze_js_star_STAR_.invokeStatic(analyzer.cljc:3028)
        at cljs.analyzer$analyze_js_star_STAR_.invoke(analyzer.cljc:3026)
        at cljs.analyzer$analyze_js_star.invokeStatic(analyzer.cljc:3056)
        at cljs.analyzer$analyze_js_star.invoke(analyzer.cljc:3055)
        at cljs.analyzer$eval3087$fn__3089.invoke(analyzer.cljc:3063)
        at clojure.lang.MultiFn.invoke(MultiFn.java:251)
        at cljs.analyzer$analyze_seq_STAR_.invokeStatic(analyzer.cljc:3339)
        at cljs.analyzer$analyze_seq_STAR_.invoke(analyzer.cljc:3337)
        at cljs.analyzer$analyze_seq_STAR__wrap.invokeStatic(analyzer.cljc:3344)
        at cljs.analyzer$analyze_seq_STAR__wrap.invoke(analyzer.cljc:3342)
        at cljs.analyzer$analyze_seq.invokeStatic(analyzer.cljc:3368)
        at cljs.analyzer$analyze_seq.invoke(analyzer.cljc:3346)
        at cljs.analyzer$analyze_form.invokeStatic(analyzer.cljc:3536)
        at cljs.analyzer$analyze_form.invoke(analyzer.cljc:3532)
        at cljs.analyzer$analyze_STAR_.invokeStatic(analyzer.cljc:3585)
        at cljs.analyzer$analyze_STAR_.invoke(analyzer.cljc:3576)
        at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:3610)
        at cljs.analyzer$analyze.invoke(analyzer.cljc:3593)
        at cljs.analyzer$analyze_seq.invokeStatic(analyzer.cljc:3369)
        at cljs.analyzer$analyze_seq.invoke(analyzer.cljc:3346)
        at cljs.analyzer$analyze_form.invokeStatic(analyzer.cljc:3536)
        at cljs.analyzer$analyze_form.invoke(analyzer.cljc:3532)
        at cljs.analyzer$analyze_STAR_.invokeStatic(analyzer.cljc:3585)
        at cljs.analyzer$analyze_STAR_.invoke(analyzer.cljc:3576)
        at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:3610)
        at cljs.analyzer$analyze.invoke(analyzer.cljc:3593)
        at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:3602)
        at cljs.analyzer$analyze.invoke(analyzer.cljc:3593)
        at cljs.analyzer$eval2339$fn__2340$fn__2343.invoke(analyzer.cljc:1539)
        at cljs.analyzer$eval2339$fn__2340.invoke(analyzer.cljc:1537)
        at clojure.lang.MultiFn.invoke(MultiFn.java:251)
        at cljs.analyzer$analyze_seq_STAR_.invokeStatic(analyzer.cljc:3339)
        at cljs.analyzer$analyze_seq_STAR_.invoke(analyzer.cljc:3337)
        at cljs.analyzer$analyze_seq_STAR__wrap.invokeStatic(analyzer.cljc:3344)
        at cljs.analyzer$analyze_seq_STAR__wrap.invoke(analyzer.cljc:3342)
        at cljs.analyzer$analyze_seq.invokeStatic(analyzer.cljc:3368)
        at cljs.analyzer$analyze_seq.invoke(analyzer.cljc:3346)
        at cljs.analyzer$analyze_form.invokeStatic(analyzer.cljc:3536)
        at cljs.analyzer$analyze_form.invoke(analyzer.cljc:3532)
        at cljs.analyzer$analyze_STAR_.invokeStatic(analyzer.cljc:3585)
        at cljs.analyzer$analyze_STAR_.invoke(analyzer.cljc:3576)
        at cljs.analyzer$analyze.invokeStatic(analyzer.cljc:3610)
        at cljs.analyzer$analyze.invoke(analyzer.cljc:3593)
        at cljs.compiler$emit_source.invokeStatic(compiler.cljc:1377)
        at cljs.compiler$emit_source.invoke(compiler.cljc:1356)
        at cljs.compiler$compile_file_STAR_$fn__4554.invoke(compiler.cljc:1456)
        at cljs.compiler$with_core_cljs.invokeStatic(compiler.cljc:1271)
        at cljs.compiler$with_core_cljs.invoke(compiler.cljc:1260)
        at cljs.compiler$compile_file_STAR_.invokeStatic(compiler.cljc:1441)
        at cljs.compiler$compile_file_STAR_.invoke(compiler.cljc:1434)
        at cljs.compiler$compile_file$fn__4585.invoke(compiler.cljc:1538)
        ... 40 more
Caused by: java.io.FileNotFoundException: node_modules\web3\dist\web3.min.js (The system cannot find the path specified)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at clojure.java.io$fn__10968.invokeStatic(io.clj:229)
        at clojure.java.io$fn__10968.invoke(io.clj:229)
        at clojure.java.io$fn__10881$G__10874__10888.invoke(io.clj:69)
        at clojure.java.io$fn__10980.invokeStatic(io.clj:258)
        at clojure.java.io$fn__10980.invoke(io.clj:254)
        at clojure.java.io$fn__10881$G__10874__10888.invoke(io.clj:69)
        at clojure.java.io$fn__10942.invokeStatic(io.clj:165)
        at clojure.java.io$fn__10942.invoke(io.clj:165)
        at clojure.java.io$fn__10894$G__10870__10901.invoke(io.clj:69)
        at clojure.java.io$reader.invokeStatic(io.clj:102)
        at clojure.java.io$reader.doInvoke(io.clj:86)
        at clojure.lang.RestFn.invoke(RestFn.java:410)
        at clojure.lang.AFn.applyToHelper(AFn.java:154)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.core$apply.invokeStatic(core.clj:659)
        at clojure.core$slurp.invokeStatic(core.clj:6862)
        at clojure.core$slurp.doInvoke(core.clj:6862)
        at clojure.lang.RestFn.invoke(RestFn.java:410)
        at status_im.utils.slurp$slurp.invokeStatic(slurp.clj:7)
        at status_im.utils.slurp$slurp.invoke(slurp.clj:6)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.core$apply.invokeStatic(core.clj:661)
        at clojure.core$apply.invoke(core.clj:652)
        at cljs.analyzer$macroexpand_1_STAR_$fn__3151.invoke(analyzer.cljc:3286)
        at cljs.analyzer$macroexpand_1_STAR_.invokeStatic(analyzer.cljc:3285)
        at cljs.analyzer$macroexpand_1_STAR_.invoke(analyzer.cljc:3271)
        ... 114 more
```
Do you have any idea what's wrong?

**Comment by fox-daiki-yamamoto** [comment]
@andytudhope I have not joined the chat. Because I'm not good at English. I am using Google translation again this time

**Comment by fox-daiki-yamamoto** [comment]
@dumiz 
I am an engineer, but I do not know if I am qualified as a reviewer. I pointed out that Japanese was wrong.

↑のエラーですが
「node_modules/web3/dist/web3.min.js」
There is an error saying that you can not find the path ↑
Did you do the command of「npm install」?

**Comment by fox-daiki-yamamoto** [comment]
@dumiz 
I am an engineer, but I do not know if I am qualified as a reviewer. I pointed out that Japanese was wrong.

It is an error of ↑
「node_modules/web3/dist/web3.min.js」
There is an error saying that you can not find the path ↑
Did you do the command of「npm install」?

**Comment by andytudhope** [comment]
No problem on the Google translate stuff @fox-daiki-yamamoto  - we just like to send a small reward to people who review translations, so we need your ETH address and it is probably better not to post it publicly.  

@dumiz that error is because you need to type `npm install` (and may need to do it twice right now because of a regression). Don't stress though, as long as the `:ja` key is added to `i18n.cljs` and is aligned properly, that's all we need 👍

**Comment by fox-daiki-yamamoto** [comment]
@andytudhope 
OK. I joined the chat earlier.

**Comment by dumiz** [comment]
@andytudhope sorry, I don't have a Linux machine, so when i try to run `npm install` it gives me error about failed running on some shell script.
I will try to find a linux machine to run it later today.

**Comment by andytudhope** [comment]
@fox-daiki-yamamoto do you think that all of the translations are now correct? Can we think about merging this PR?
@dumiz please check one more time that you are totally happy with each translation?

File: src/status_im/translations/ja.cljs [hunk 14]
```diff
@@ -158,83 +177,204 @@
...
    :from                                  "送信元"
    :data                                  "データ"
    :got-it                                "理解しました"
+   :block                                 "ブロック"
+   :hash                                  "ハッシュ"
+   :gas-limit                             "gas制限"
+   :gas-price                             "gas価格"
+   :gas-used                              "使用Gas"
+   :cost-fee                              "費用/料金"
+   :nonce                                 "現在時間"
```
Comment (dumiz) [inline src/status_im/translations/ja.cljs]: @andytudhope What does "nonce" mean here?

File: src/status_im/translations/ja.cljs [hunk 15]
```diff
@@ -158,83 +177,204 @@
...
    :delete-group-prompt                   "連絡先には影響を与えません。"
    :contact-s                             {:one   "連絡先"
                                            :other "連絡先"}
-   ;participants
 
-   ;protocol
+   ;;protocol
    :received-invitation                   "チャット招待状を受信しました"
    :removed-from-chat                     "グループチャットから削除されました"
    :left                                  "残り"
```
Comment (dumiz) [inline src/status_im/translations/ja.cljs]: The word "left" here means someone left the chat right?

Comment (dumiz) [inline src/status_im/translations/ja.cljs]: @andytudhope The word "left" here means someone left the chat, right?

File: src/status_im/translations/ja.cljs (hunk 14 above)
Comment (dumiz) [inline src/status_im/translations/ja.cljs]: @andytudhope What does "nonce" mean here?

File: src/status_im/translations/ja.cljs (hunk 15 above)
Comment (andytudhope) [reply src/status_im/translations/ja.cljs]: Yeah, that is correct.

File: src/status_im/translations/ja.cljs (hunk 14 above)
Comment (andytudhope) [reply src/status_im/translations/ja.cljs]: Nonce is a technical word that refers to a unique number used to identify transactions: https://en.wikipedia.org/wiki/Cryptographic_nonce

**Comment by dumiz** [comment]
@andytudhope The translation looks good to me now.

**Comment by oskarth** [comment]
@andytudhope @fox-daiki-yamamoto What's the state of this PR?

**Review by andytudhope — APPROVED** [review]

**Comment by andytudhope** [comment]
@dumiz please squash your commits and I will merge this and award the bounty, thank you very much! 🎉

**Comment by dumiz** [comment]
@andytudhope No problem.

**Pull request merged**